src/style.css

```
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  width: 100px;
+  height: 16px;
+  justify-content: flex-start;
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  width: 16px;
+  margin-right: 4px;
+  text-align: center;
+}
+
+.stars--5 > .stars__star,
+.stars--4 > .stars__star:nth-child(-n + 4),
+.stars--3 > .stars__star:nth-child(-n + 3),
+.stars--2 > .stars__star:nth-child(-n + 2),
```

```suggestion
.stars--4 > .stars__star:nth-child(-n+4),
.stars--3 > .stars__star:nth-child(-n+3),
.stars--2 > .stars__star:nth-child(-n+2),
```
In css those space are not neccessary